webapp/vue-portlet/src/main/webapp/common/components/ExoModal.vue

```diff
@@ -0,0 +1,58 @@
+<template>
+  <div v-show="show" ref="modal" class="modal-mask uiPopupWrapper">
+    <div class="uiPopup modal-content">
+      <div class="popupHeader">
+        <a class="uiIconClose pull-right" @click="$emit('close')"></a>
+        <span class="popupTitle">{{ title }}</span>
+      </div>
+      <div class="PopupContent popupContent">
+        <slot></slot>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+const EVEN = 2;
+export default {
+  props: {
+    title: {
+      type: String,
+      default: ''
+    },
+    show: {
+      type: Boolean,
+      default: false
+    }
+  },
+  watch: {
+    show() {
+      if (this.show) {
+        const escapeKeyCode = 27;
+        document.onkeyup = evt => {
+          evt = evt || window.event;
+          if (evt.keyCode === escapeKeyCode) {
+            this.$emit('close');
+          }
+        };
+
+        //this is workaround to fix chrome bug
+        //popup is blurring when using translate() and the size is odd
+        Vue.nextTick(function() {
+          const width = this.$el.offsetWidth;
```

Comment:
IMO, it's better to use Vuetify that manages perfectly all of those events, you can take a look at : 
https://github.com/Meeds-io/social/blob/ee43b9d23779803bfcc2900c69dbc70432a1fe01/webapp/vue-portlet/src/main/webapp/common/components/ExoConfirmDialog.vue

that is a modal for 'confirmation'

Reply:
it's done!